docs/tutorials/model-registry.txt

```diff
@@ -0,0 +1,245 @@
+the Swagger API documentation.
+
+The following snippet creates a model and returns a
+:class:`~determined.experimental.Model`. The
+:class:`~determined.experimental.Model` is covered below.
+
+.. code:: Python
+
+  from determined.experimental import Determined
+
+  model = Determined().create_model(
+            "model_name",
+            description="optional description",
+            metadata={"optional": "JSON serializable dictionary"}
+          )
+
+Similarly, you can create a model from the CLI using the following command.
+
+.. code:: bash
+
+  det model create <model_name> description=""
+
+
+The following snippet returns models registered in Determined as a list of
+:class:`~determined.experimental.Model` objects. Models can be sorted by name,
+description, creation time and last updated time. Additionally, models can be
+filtered by name or description via the Python API. For sorting and ordering
+options see :class:`~determined.experimental.ModelSortBy` and
+:class:`~determined.experimental.ModelOrderBy` respectively.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  d = Determined()
+
+  all_models = d.get_models()
+
+  chronological_sort = d.get_models(sort_by=ModelSortBy.CREATION_TIME)
+
+  # The following will return all models with "mnist" in their name. Some
+  # potential model names are "mnist_pytorch", "mnist_cnn", "mnist", etc.
+  mnist_models = d.get_models(name="mnist")
+
+  # Similarly models can be filtered by description. The following line returns
+  # models whose description contains "ocr".
+  ocr_models = d.get_models(description="ocr")
+
+
+Similarly, you can list models from the CLI using the following command.
+
+.. code:: bash
+
+   det model list --sort-by={name,description,creation_time,last_updated_time} --order-by={asc, desc}
+
+
+The following snippet queries a model by name.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  model = Determined().get_model("model_name")
+
+The CLI equivalent is below. The describe command will print the information of
+the latest version of the model by default as well.
+
+.. code:: bash
+
+  det model describe <model_name>
+
+Currently, metadata from models can only be edited via the Python API. The
+methods are as follows.
+
+.. code:: python
+
+  from determined.experimental import Determined
+
+  model = Determined().get_model("model_name")
+
+  model.add_metadata({"key", "value"})
+
+  # Metadata is merged with existing metadata.
+  model.add_metadata({"metrics": {"test_set_loss": 0.091}})
+  # results in {"key": "value", "metrics": { "test_set_loss": 0.091 } }
+
+  # Only top-level keys are merged. The following line will override all values
+  # underneath the "metrics" key.
+  model.add_metadata({"metrics": {"test_set_acc": 0.97}})
+  # results in {"key": "value", "metrics": { "test_set_acc": 0.97 } }
+
+  # removes a top level key from the metadata
```

Comment:
For consistency, capitalize the first letter of the comment and punctuate with a period.